src/Middleware/MaintenanceMiddleware.php

```diff
@@ -55,6 +62,7 @@ class MaintenanceMiddleware
     public function __construct($config = [])
     {
         $this->config($config);
+        $this->whitelist = Configure::read('Wrench.whitelist');
```

Comment:
What about having the whitelist in the `MaintenantMiddleware` `__construct` method instead of having it being passed as a Configure value ?

So people can do something like :

```php
$middleware->add(new MaintenanceMiddleware(['whiteList' => ['127.0.0.1']));
```

What do you think ?